napari/_vispy/layers/base.py

```diff
@@ -222,7 +222,12 @@ def _on_matrix_change(self):
         ):
             # The last downsample factor is used because we only ever show the
             # last/lowest multi-scale level for 3D.
-            translate += (self.layer.downsample_factors[-1][::-1] - 1) / 2
+            translate += (
+                self.layer.downsample_factors[-1][
+                    self.layer._slice_input.displayed
+                ][::-1]  # displayed dimensions, invert to match VisPy order
+                - 1
+            ) / 2
```

Comment:
Black doing its awful thing once again. Maybe like this is more readable?

```suggestion
			down_fac = self.layer.downsample_factors[-1]
			dims_displayed = self.layer._slice_input.displayed
			# invert to match VisPy order
            translate += (down_fac[dims_displayed][::-1] - 1) / 2
```

Reply:
It is not black, it is ruff this time

Reply:
Yeah, but it's the black spec right?

Reply:
Another option to fight the auto-formatters is to pull `self.layer._slice_input.displayed` our as `dims_displayed` earlier in the function, since it's used a few times.

Reply:
Yeah thought the same but it was too far to suggest :P

Reply:
Addressed with @andy-sweet's suggested fix. It gets assigned further down anyway! So might as well!